backend/Infraestructure/IOC/IOCInfraestructure.cs

```diff
@@ -16,14 +18,23 @@ public static class IOCInfraestructure
     {
         public static void AddPersistence(this IServiceCollection services, IConfiguration configuration)
         {
+            #region Context
             services.AddDbContext<ApplicationDbContext>(options =>
             {
                 options.UseSqlite(configuration.GetConnectionString("SQLiteConnection"), b => b.MigrationsAssembly("Infraestructure.IOC"));
             });
+            #endregion
 
-            services.AddIdentity<IdentityUser, IdentityRole>()
+            #region Identity
+            services.AddIdentity<Admin, IdentityRole>()
                      .AddEntityFrameworkStores<ApplicationDbContext>()
                      .AddDefaultTokenProviders();
+            #endregion
+
+            #region Repositories
+            services.AddTransient<IAdminRepository, AdminRepository>();
```

Comment:
Since this is a Repository, even if we are only adding this once, I dont think we want this to be created each time is called, AddScoped I think fits this case better than AddTransient.